app/root.tsx

```diff
@@ -74,13 +75,13 @@ function LayoutContent({ children }: { children: React.ReactNode }) {
         <Meta />
         <Links />
       </head>
-      <body>
+      <Page>
```

Comment:
is the only benefit of this that on each route component now you can avoid writing `<Page>`? If yes then imo it doesn't give us much for the cost being zero flexibility (every route having to use page component now)

Reply:
Yea and makes it easier to not have a div wrapping the header, main, and footer that is inside the body tag; not sure how much that matters. Would you rather that each route that renders a page is where we use this component?

Reply:
yeah I don't think that div matters and yes I would rather have each route picking if they want to use page or not

Reply:
sometimes a bit more boilerplate is healthier

Reply:
reverted and added the page component to your auth routes